src/pages/options/options.js

```diff
@@ -199,80 +199,74 @@ const OptionsApp = () => {
   const loginAction = () => openTabWithUrl(AUTH_URL)
 
   return (
-    <OptionsContainer>
-      <OptionsTitle darkMode={darkMode}>
-        {PocketLogo({
-          width: '26px',
-          height: '26px',
-          marginRight: '10px',
-          marginTop: '3px',
-          verticalAlign: 'text-top'
-        })}
+    <div className={optionsContainer}>
+      <h1 className={cx(optionsTitle, darkMode ? 'darkMode' : null)}>
         {localize('options_page', 'header')} -{' '}
         {localize('options_page', 'save_to_pocket')}
-      </OptionsTitle>
+      </h1>
 
-      <OptionsSection>
-        <OptionsSectionContent>
-          <OptionsSectionTitle>
+      <div className={optionsSection}>
+        <div className={optionsSectionContent}>
+          <div className={cx(optionsSectionTitle, darkMode ? 'darkMode' : null)}>
             {localize('options_page', 'login_title')}
-          </OptionsSectionTitle>
-          <OptionsSectionMain>
-            {(accessToken && userName) ? (
+          </div>
+          <div className={cx(optionsSectionMain, darkMode ? 'darkMode' : null)}>
+            TODO: fix accessToken & userName stuff here
+            {/* {(accessToken && userName) ? (
               <div>
                 {userName} &nbsp; ({' '}
-                <OptionsButtonLink onClick={logoutAction}>
+                <button className={optionsButtonLink} onClick={logoutAction}>
                   {localize('options_page', 'logout_link')}
-                </OptionsButtonLink>{' '})
+                </button>{' '})
               </div>
             ) : (
-              <OptionsButtonLink onClick={loginAction}>
+              <button className={optionsButtonLink} onClick={loginAction}>
                 {localize('options_page', 'login_link')}
-              </OptionsButtonLink>
-            )}
-          </OptionsSectionMain>
-        </OptionsSectionContent>
-      </OptionsSection>
+              </button>
+            )} */}
```

Comment:
This is the section that will need to be readdressed after I figure out what's up with getting the token & username